orchestra/models/core/models.py

```diff
@@ -576,6 +577,34 @@ class PayRate(PayRateMixin, models.Model):
     end_date = models.DateField(null=True, blank=True)
 
 
+class ChecklistTemplate (ChecklistTemplateMixin, BaseModel):
+    """
+    A checklist template
+
+    Attributes:
+        slug (str):
+            The string that use to refer the checklist
+        name (str):
+            The name of the checklist
+        description (str):
+            A text description of the checklist
+        creator (orchestra.models.Worker)
+            The worker who creates the checklist. This field is null
+            if it is generated by the system.
+        todos (str)
+            A JSON blob that describe the todos in the checklist
+    """
+    class Meta:
+        app_label = 'orchestra'
+
+    slug = models.CharField(max_length=200, unique=True)
+    name = models.CharField(max_length=200)
+    description = models.TextField()
+    creator = models.ForeignKey(
+        Worker, null=True, related_name='creator', on_delete=models.CASCADE)
```

Comment:
should the delete SET_NULL so we don't lose the template?

Reply:
@marcua Shouldn't manage different versions of a template (add a version column?). I suppose we are going to update templates over time.

Reply:
I'm pro-versioning, though I think it's too early for the needs of the product right now. I still think we should SET_NULL to be safe for the time being.